models/project_board.go

```diff
@@ -187,34 +190,63 @@ func GetUncategorizedBoard(projectID int64) (*ProjectBoard, error) {
 	}, nil
 }
 
-// LoadIssues load issues assigned to this board
-func (b *ProjectBoard) LoadIssues() (IssueList, error) {
+// LoadProjectIssues load project issues assigned to this board
+func (b *ProjectBoard) LoadProjectIssues() ([]*ProjectIssue, error) {
 	var boardID int64
 	if !b.Default {
 		boardID = b.ID
 
 	} else {
 		// Issues without ProjectBoardID
-		boardID = -1
+		boardID = 0
 	}
-	issues, err := Issues(&IssuesOptions{
-		ProjectBoardID: boardID,
-		ProjectID:      b.ProjectID,
-	})
-	b.Issues = issues
-	return issues, err
+	var projectIssues []*ProjectIssue
+	var issues []*Issue
+	if err := x.Table("issue").
+		Cols("issue.id, issue.repo_id, issue.index, issue.poster_id,issue.original_author,issue.original_author_id,issue.name,issue.content,issue.milestone_id,issue.priority,issue.is_closed,issue.is_pull,issue.num_comments,issue.ref,issue.deadline_unix,issue.created_unix,issue.updated_unix,issue.closed_unix,issue.is_locked, project_issue.id as project_issue_id, project_issue.project_id as project_issue_project_id, project_issue.project_board_id as project_issue_project_board_id").
```

Comment:
We absolutely cannot merge with this. It's an unmaintainable nightmare.

Reply:
as I said, your suggestion doesn't work with xorm so if you have something better in mind feel free to suggest :)

Reply:
I have submitted PR to xorm to fix issue with `` `issue`.* `` quoting

Reply:
ok, I've tried @lafriks patch and it works great. I've pushed a fix with the updated query but still waiting for a new version of xorm to be tagged to update go.mod by @lunny.